### PR TITLE
chore(i18n): remove classic theme - FE-3111

### PR DESCRIPTION
**Pull request by DipperTheDan**
removal of classic story and redundant docs. Stories have been refactored in to CSF and an MDX
example created.

fixes FE-3111

### Proposed behaviour
- Remove `classic` story
- Refactor stories file to CSF
- Create MDX example for component 
- Remove redundant documentation

### Current behaviour
- `classic` story is present
- Stories file is in the deprecated `storiesOf` format
- Documentation conforms with our previous standards

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Additional context
N/A

### Testing instructions
- `git checkout FE-3111_i18n_classic_theme_removal`
- `npm start` to start Storybook
- Navigate to i18nComponent
- Component should not have a classic example and MDX example should be present.

### CODE REVIEW

File: src/components/i18n/i18n.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+<Meta title="i18nComponent" parameters={{ info: { disable: true }}} />
+
+# I18nComponent (Internationalization)
+Displays a translation from an i18n file.
+
+Internationalization (i18n) means apps can be localized for languages and cultures easily.
+This component displays a single text translation.
+You can use this component with any other component that displays text.
+
```
Comment (jamime) [inline src/components/i18n/i18n.stories.mdx:13]: I think we should highlight that we don't reccomend using this component any more.

Please can you add a warning using the `DeprecationWarning` component.

Something along the lines of 

> It's on our roadmap to remove our I18n implementation and provide localisation for our components out of the box. If you're adding localisation to your application we reccomend you use a dedicated solution such as `react-i18next`

**Review by jamime — CHANGES_REQUESTED** [review]

File: src/components/i18n/i18n.stories.mdx (hunk 1 above)
Comment (DipperTheDan) [reply src/components/i18n/i18n.stories.mdx:20]: I've added the deprecation warning as requested 😄

**Review by edleeks87 — APPROVED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

File: src/components/i18n/i18n.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      }; 
+      return (
+        <I18nComponent markdown scope='my.example'/>
+      )         
+    }}
+  </Story>
+</Preview>
+
+### Inline
+<Preview>
```
Comment (DlgSHi) [inline src/components/i18n/i18n.stories.mdx:61]: there is no change between `default` and `with Inline` so we need to ignore `chromatic` for this story

Comment (DipperTheDan) [reply src/components/i18n/i18n.stories.mdx:61]: I've added the parameter to the story to prevent a chromatic snapshot being taken 😄

**Review by DlgSHi — APPROVED** [review]

**Review by nicktitchmarsh — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Pull request merged**